src/workarounds/forwarddiff_rules.jl

```diff
@@ -0,0 +1,161 @@
+import ForwardDiff
+import AbstractFFTs
+
+# original PR by mcabbott: https://github.com/JuliaDiff/ForwardDiff.jl/pull/495
+# modified version: https://github.com/niklasschmitz/ForwardDiff.jl/blob/nfs/fft/src/fft.jl
```

Comment:
Is this comment useful (in the sense: Will this not point to void when the branch is merged / deleted)

Reply:
Agree, I removed the pointer to my branch